sale_delivery_block/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Sale Delivery Block",
+    "summary": "Allow you to block the creation of deliveries "
+               "from a sale order.",
+    "version": "10.0.1.0.0",
+    "author": "Eficent, "
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/sale-workflow",
+    "category": "Sales",
+    "depends": ["sale_stock"],
```

Comment:
IMHO it's better to define explicit dependencies (at first sight I saw base and sale). It facilitates understanding module aim and migration in case of changes in dependant modules.

Reply:
I recently had a case with exactly the same dependency. It depended on 'sale' and 'stock' instead of 'sale_stock' and it broke the tests as they were executed before the glue module was loaded.

Reply:
I think that let dependencies holes. Moreover, I think building dependencies on tests failures (or not) is a bad idea. But, just my opinion

Reply:
Sorry, I don't understand that. Can you clarify the exact problem? I think the test failure simply indicates that sale_stock is the proper dependency because Odoo is not designed to have sale and stock installed but not sale_stock.

Reply:
I see. I just say that explicit depends is better than implicit.

If your module depends on [logic, data, ..] defined on another, take it (to summarize, if you have to depend on 'sale_stock', take it, but if you depend on sale and stock, take them both too)

Reply:
According to this almost every module should depend on base. I have no idea if putting all the implicit dependencies to every module would lead to other problems.